src/pages/home/report/ReportActionsView.js

```diff
@@ -76,29 +76,28 @@ class ReportActionsView extends React.Component {
         // Helper variable that keeps track of the unread action count before it updates to zero
         this.unreadActionCount = 0;
 
+        // Todo: this should come from props and depend on isDrawerOutOfTheWay
         // Helper variable that prevents the unread indicator to show up for new messages
         // received while the report is still active
         this.shouldShowUnreadActionIndicator = true;
```

Comment:
On most chats that I've used you'll still see new messages split line even if the chat is still active, the line will disappear when you interact with the chat

Reply:
This is getting refactored in another open PR. Which consequently is jumping through a lot of hoops to figure out when the report has been changed (which I believe this PR will simplify).

https://github.com/Expensify/Expensify.cash/pull/1977

Reply:
Oki, what I meant was that at least the initial value could come from outside, because on mobile you can still have an active chat covered by the drawer, so you'd want to see an indicator of where new messages start from when you collapse the drawer

Reply:
>so you'd want to see an indicator of where new messages start from when you collapse the drawer

The PR I linked changes the general behavior of this marker. I'm not sure there has been any discussion about what to do if the drawer is closed or not. But I also think maybe we don't need to worry about this right now?